.storybook/theme.js

```diff
@@ -27,5 +27,5 @@ export default create({
   // Branding
   brandTitle: 'Vue Vocabulary',
   brandUrl: 'https://opensource.creativecommons.org/cc-vue-vocabulary',
-  brandImage: 'https://raw.githubusercontent.com/creativecommons/vue-vocabulary/master/readme_assets/vocabulary_logo.svg?sanitize=true'
+  brandImage: 'logos/products/vocabulary.svg#vocabulary'
```

Comment:
This won't work. It works in Vocabulary because it contains the actual SVGs, but Vue Vocabulary does not.

Reply:
I added the svg to this repo, so it worked. But I guess I didn't have to do it as we are using `vocabulary` as SSOT (as you mentioned).